solutions/swb-ui/pnpm-lock.yaml

```diff
@@ -1,71 +0,0 @@
-lockfileVersion: 5.3
```

Comment:
Is this file supposed to be deleted? Shouldn't there still be a `pnpm-lock.yaml` file?

Reply:
None of our other (non-common) packages have a lock file, trying to stay consistent.